build.gradle

```diff
@@ -40,8 +40,8 @@ subprojects {
         mavenCentral()
     }
 
-    sourceCompatibility = 1.8
-    targetCompatibility = 1.8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Is there any reason don't upgrade to version 17? Because in test.yml file it already defined as 17!

Reply:
I will use 17 everywhere